site/src/routes/embedding-knn/AgreementClusterStaticPlot.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  export let data: any[] = [];
+
+  var names = data.map((d) => d["ego_primary_label"]);
+  names = names.slice(0, 20);
+  var selected: string = "normoc";
```

Comment:
Nit: I recommend using `let` over `var` whenever you can (it's cleaner that way, in the way that scheme is more sanitary than common lisp). `let` is lexically scoped and can only be declared once.

https://stackoverflow.com/questions/762011/what-is-the-difference-between-let-and-var